apps/src/music/views/ImageAttributions.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+
+import musicI18n from '../locale';
+import {ImageAttributionCopyright} from '../player/MusicLibrary';
+
+import moduleStyles from './ImageAttributions.module.scss';
+
+interface ImageAttributionsProps {
+  attributions: ImageAttributionCopyright[];
+}
+
+const creativeCommonsLicenseLinks: {
+  [version: string]: {name: string; url: string};
```

Comment:
Optional; if you create a type for `licenseVersion`, you can make this object a `{[version in LicenseVersion]?: {name: string; url: string}}` to ensure the keys match up

Reply:
Nice!  Updated.